src/screens/AddParticipant/index.js

```diff
@@ -234,7 +234,7 @@ export default AddParticipant;
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the container might significantly alter the appearance of the screen. Ensure this color change aligns with the overall design and accessibility guidelines, particularly for readability.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->